bool enable
---
uint8 status